extract/extract/extractors/zalando.py

```diff
@@ -180,19 +175,19 @@ def _get_sustainability(
             beautiful_soup=beautiful_soup,
             title_attr="certificate__title",
             description_attr="certificate__description",
```

Comment:
This wasn't part of the issue but I think it would be better to create a parameter of `_get_sustainability` with these strings as default values and use them here. Could you change this as well?

Reply:
Do you mean like this:

```
def _get_sustainability(
    beautiful_soup: BeautifulSoup,
    certificate_attr: str = "cluster-certificates",
    labels_title_attr: str = "certificate__title",
    labels_description_attr: str = "certificate__description",
    impact_title_attr: str = "cluster-causes__title",
    impact_description_attr: str = "cluster-causes__intro-statement",
    aspects_title_attr: str = "cause__title",
    aspects_description_attr: str = "cause__description",
) -> List[str]:

Reply:
You shouldn't resolve comments if they aren't ;)

Reply:
Oh, I'm sorry, didn't recognize that they are not the same! I skimmed through and thought they are always the same..

You can ignore this comment. I think sticking to the single parameter `certificate_attr: str = "cluster-certificates"` is perfectly fine!

Reply:
Ok, so I won't change anything regarding this.